target_chains/solana/examples/send_usd/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "my_first_pyth_app"
```

Comment:
mmmm maybe rename this every to send_usd?

Reply:
done